common/v2/services/EthService/wallet/deterministic/mnemonic.ts

```diff
@@ -1,6 +1,7 @@
-import { decryptMnemonicToPrivKey } from 'libs/decrypt';
 import { fromPrivateKey } from 'ethereumjs-wallet';
+
 import { signWrapper } from 'libs/wallet';
+import { decryptMnemonicToPrivKey } from '../../utils';
```

Comment:
Consider writing this as `v2/utils`

Reply:
I'm intentionally placing all Eth utils in a specific folder with the ultimate goal of remove all chain related manipulations out of the components, and as much as possible into `EthService`.
By keeping them separate from the other `utils` I hope it can help build more confidence in the code base, be easier to reach full coverage, or be simpler to switch to another library such as `etherjs`.

Reply:
Changed to `v2/services/EthService/utils`